docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -36,20 +39,115 @@ To define an AMQP broker connection, add an `<amqp-connection>` element within t
 
 *Notice:* If you disable auto-start on the broker connection, the start of the broker connection will only happen after the management method `startBrokerConnection(connectionName)` is called on the ServerController.
 
-*Important*: The target endpoint needs permission for all operations that you configure. Therefore, If you are using a security manager, ensure that you perform the configured operations as a user with sufficient permissions.
+*Important*: In addition to a matching mirror ID, the target endpoint needs permission for all operations that you configure. Therefore, If you are using a security manager, ensure that you perform the configured operations as a user with sufficient permissions.
```

Comment:
this will be updated.. I removed some code that needed matching between source and target.


I'm still updating docs.. docs are not updated yet. (hence that's why it's a draft)